almdrlib/client.py

```diff
@@ -363,7 +378,7 @@ def schema_name(self):
 
     @property
     def required(self):
-        return self._required
+        return self._required or self._in == OpenAPIKeyWord.PATH
```

Comment:
Path parameters are always required in OpenAPI.  Even if alsdkdefs doesn't mark them that way in the specs.  (Perhaps that should be validated...)